Slim/Web/HTTP.pm

```diff
@@ -1511,6 +1511,9 @@ sub generateHTTPResponse {
 		# $peerclient as this will cause streaming to the real client $client to stop.
 		delete $peerclient{$httpClient};
 
+		# Disable metadata in case this client sent an Icy-Metadata header
```

Comment:
I guess you copied/pasted this comment from line 1694. But how is this conditional ("in case...")? Would you know where that case is checked? Just asking...

Reply:
AFAIU, that case should never be checked when sending a "FileHandle", exactly like when we are doing a SendStreamingFile. It's just a safeguard there in case of faulty (or unusual) GET request and I'm extending that safeguard. Now we could use another approach in SendStreamingResponse, ~2328 and add a test on $client 
```
if ($client && $sendMetaData{$httpClient}) {
```
Here I'm 100% sure that we should never enter that code when there is no client. Now, I think option \#1 if the more "consistent" with the rest of the code, but I can wrong (would be first, wouldn't it? :smile:)